api/src/main/java/jakarta/servlet/http/HttpServlet.java

```diff
@@ -78,6 +80,7 @@ public abstract class HttpServlet extends GenericServlet {
 
     private static final String HEADER_IFMODSINCE = "If-Modified-Since";
     private static final String HEADER_LASTMOD = "Last-Modified";
+    private static final List<String> HEADER_SENSITIVE = Arrays.asList("Authorization", "Accept");
```

Comment:
Why these two methods? `Authorization` I understand but not `Accept`. And why not `Cookie`?

Reply:
I agree that `Cookie` is a sensitive header, whilst no so much `Accept`.

I also think that any proxy headers (`X-Forwarded-*`, `Forwarded`,  `Proxy-Authorization`) should be protected so that trace can't be used to probe the internals of a data centre.

Reply:
Then I will remove the class field HEADER_SENSITIVE and move all implementation to isSensitiveHeader method